comprobo_final_project/scripts/robot_controller.py

```diff
@@ -88,15 +82,14 @@ def run(self, duration):
 
 
 if __name__ == '__main__':
-
     genes = [0.0, 1.0, 1.0, 0.0]
-    duration = 15
+
     robot = Robot()
     robot.pose.position.x = 3.0
     robot.pose.position.y = 5.0
-    simulator = Simulator(robot, True)
-    simulator.render()
-    robot_controller = RobotController(robot, genes, simulator)
+
+    robot_controller = RobotController(robot, genes)
+    simulation_visualizer = SimulationVisualizer(robot)
```

Comment:
how is this made? will simulation visualizer now always have to be called for every simulation? not sure about the interaction between codes